[
  {
    "message": "Cannot use continue outside a loop",
    "line": 1
  }
]